pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -171,13 +171,8 @@ public ServerType getServerType() {
     public static final String CONFIG_OF_REQUEST_HANDLER_FACTORY_CLASS = "pinot.server.requestHandlerFactory.class";
     public static final String CONFIG_OF_NETTY_PORT = "pinot.server.netty.port";
     public static final String CONFIG_OF_ADMIN_API_PORT = "pinot.server.adminapi.port";
-    public static final String CONFIG_OF_STARTER_ENABLE_SEGMENTS_LOADING_CHECK =
```

Comment:
Can we log these configs as `deprecated` in the server log when startup to notify these configs have been deprecated? Otherwise, it'll be hard for open source to notice this changes.

Reply:
IMO we should not keep unused constants in the code base, it's better to put them in the release log.

Reply:
Discussed offline, log warnings for usage of deprecated config keys